pkg/controller/controller_crd_event_handler.go

```diff
@@ -25,7 +25,8 @@ type watchState struct {
 type crdEventHandler struct {
 	ctx context.Context
 	*BundleController
-	watchers map[string]watchState // CRD name -> state
+	watchers  map[string]watchState // CRD name -> state
+	namespace string
```

Comment:
This is not necessary - `crdEventHandler` struct embeds `*BundleController` which has the field.